crates/fluvio-auth/src/error.rs

```diff
@@ -4,14 +4,14 @@ use thiserror::Error;
 /// Possible errors from Auth
 #[derive(Error, Debug)]
 pub enum AuthError {
-    #[error("IoError")]
+    #[error("IoError: {0}")]
```

Comment:
let's get rid of this and move to anyhow

Reply:
I want to do anyhow, but then it requires changes in other places in the code. Want to make this change like this and do other PRs for anyhow in future